components/form/ChangePassword.vue

```diff
@@ -41,7 +41,7 @@ export default {
     return {
       username:                    '',
       errorMessages:               [],
-      pCanShowMissmatchedPassword: false,
+      pCanShowMismatchedPassword: false,
       pIsRandomGenerated:            false,
```

Comment:
Sometimes I'd adjust whitespace, but since `pIsRandomGenerated` is out of sync, I didn't.